frameworks/react-cra/src/index.ts

```diff
@@ -24,7 +24,7 @@ export function createFrameworkDefinition(): FrameworkDefinition {
 
   return {
     id: 'react-cra',
-    name: 'react',
+    name: 'React',
```

Comment:
This is the cause of https://github.com/TanStack/create-tsrouter-app/issues/119 

We're passing in lower case 'react' into the `getFrameworkByName` function and it breaks because it's case senstive.